src/API.js

```diff
@@ -27,6 +27,18 @@ class API extends ApplicationAPI {
         Routers.SearchRouter,
         Routers.TrackingRouter,
       ],
+      v2: [
+        Routers.AlertsRouter,
+        Routers.AllStopsRouter,
```

Comment:
why not make a shared list for the shared routers, and just use the `...` (spread) operator?